.yarnrc.yml

```diff
@@ -1,4 +1,5 @@
 nodeLinker: node-modules
+checksumBehavior: update
```

Comment:
Without this, I could not run `yarn install`
I think this happens only for a new PF setup

Reply:
Generally speaking, this is a bad idea. I'm unsure whether the bug related to Yarn producing different checksums on different systems has been resolved or not. If that bug is still present, then setting it to `update` is functionally the same as `ignore` because checksum failures won't be reported (AFAIK). @pupilfirst/engineers Do you see yarn checksum failures still?